Tests/CarthageKitTests/XcodeSpec.swift

```diff
@@ -56,18 +60,22 @@ class XcodeSpec: QuickSpec {
 				expect(currentSwiftVersion?.isEmpty) == false
 			}
 
+			#if !SWIFT_PACKAGE
```

Comment:
Because SwiftPM does not build frameworks (`.framework`) so we can't test this using the existence of `Quick.framework`.